.devops/templates/build-test-lint.yml

```diff
@@ -6,6 +6,10 @@ steps:
       filePath: yarn-ci.sh
     displayName: yarn (install packages)
 
+  - script: |
+      yarn playwright install --with-deps
```

Comment:
> since Playwright is already present in master (as a dependency for Storybook).

it has nothing to do with storybook, rather storywright - VR test solution which is in completely different pipeline. this will slow down every PR by installing binaries no matter if necessary.

Reply:
> it has nothing to do with storybook, rather storywright - VR test solution which is in completely different pipeline. this will slow down every PR by installing binaries no matter the affected changes which is not something we would like to do.

Sorry for the Storybook/Storywright confusion, it was a brain typo.

I understand what you mean now. I'll remove the entry here. For `web-components-v3`, I'll change the `test` script to `e2e`, and add the `playwright install` step to the `e2e` pipeline.